quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliCreateJvmApplicationIT.java

```diff
@@ -264,6 +269,33 @@ public void verifyRestEasyReactiveAndClassicResteasyCollisionUserMsg() {
         assertBuildError(buildResult, "io.quarkus:quarkus-resteasy");
     }
 
+    @Test
+    public void devModeIgnoresPomPackaging() throws IOException {
+        QuarkusCliRestService app = cliClient.createApplication("pomApp", defaultWithFixedStream());
+        {//set packaging to POM
+            Path pom = getFileFromApplication(app, ROOT_FOLDER, "pom.xml").toPath();
+            System.out.println(pom);
+            List<String> content = Files.readAllLines(pom);
+            for (int i = 0; i < content.size(); i++) {
+                String line = content.get(i);
+                if (line.endsWith("<artifactId>pomApp</artifactId>")) {
+                    content.set(i, line + "<packaging>pom</packaging>");
+                    break;
+                }
+            }
+            Files.write(pom, content);
+        }
+        // Start using DEV mode
+        assertEquals(Duration.of(2, ChronoUnit.SECONDS),
```

Comment:
Suggestion:

`Duration.of(2, ChronoUnit.SECONDS)` -> `Duration.ofSeconds(2)`

Reply:
changed